docs/python/python-on-azure.md

```diff
@@ -18,8 +18,9 @@ The [Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.
-[Deploy Python containers to Azure App Service](https://learn.microsoft.com/azure/developer/python/tutorial-deploy-containers-01) | Deploy a Docker container using the [Docker](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) extension.
-[Deploy Python to Azure Functions](https://learn.microsoft.com/azure/azure-functions/create-first-function-vs-code-python) | Deploy serverless code using the [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) extension.
+Tutorial | Description | Related Tools
+--- | --- | ---
+[Deploy Python web app to Azure App Service](https://learn.microsoft.com/azure/app-service/quickstart-python) | Deploy a simple web app | [Django](https://www.djangoproject.com/) <br> [Flask](https://www.djangoproject.com/) <br> [Azure CLI](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azurecli) <br> [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) <br> [Azure Tools extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
+[Deploy Python web app with database to Azure App Service](https://learn.microsoft.com/azure/app-service/tutorial-python-postgresql-app) | Deploy a web app with PostgreSQL database | [Django](https://www.djangoproject.com/) <br> [Flask](https://www.djangoproject.com/) <br> [PostgreSQL](https://www.postgresql.org/download/) <br> [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) <br> [Azure Tools extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
+[Deploy Python containers to Azure App Service](https://learn.microsoft.com/azure/developer/python/tutorial-deploy-containers-01) | Deploy a Docker container |  [Docker](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) <br> [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) <br> [Azure Tools extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
+[Deploy Python to Azure Functions](https://learn.microsoft.com/azure/azure-functions/create-first-function-vs-code-python) | Deploy serverless code with Azure Functions | [Azure Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local#install-the-azure-functions-core-tools) <br> [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) <br> [Azure Tools extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
```

Comment:
I think you can leave out "extension" for "Azure Functions extension" since you aren't using "extension" in the other links.
Also this link is pointing to the Python extension rather than the Azure Functions extension.